test/test_helper.rb

```diff
@@ -25,6 +25,6 @@ def call(input, context: {}, result: {})
 # bolds any instance of the word yeH
 class YehBolderFilter < HTMLPipeline::TextFilter
   def call(input, context: {}, result: {})
-    input.gsub("yeH", "**yeH**")
+    input.gsub("yeH", "**yeH**") unless context[:no_bolding] == false
```

Comment:
nits

- I think it's more natural if `yeH` is converted only if `context[:no_bolding]` is false.
- `unless XXX == false` is difficult to read.
- We can safely remove `== true` or `== false`, and `nil` should be treated as falsy.

```suggestion
    input.gsub("yeH", "**yeH**") unless context[:no_bolding]
```

Test cases are also affected.